src/main/java/it/gov/pagopa/pu/debtpositions/config/RepositoryRestCustomConfiguration.java

```diff
@@ -44,6 +44,8 @@ public OpenApiCustomizer operationIdCustomizer() {
             )
             + (PathItem.HttpMethod.GET.equals(httpMethod) && paths.length == 3 ? "s" : "")
         ));
+        // removing duplicate schema due to ControllerExt
+        openApi.getComponents().getSchemas().remove(DebtPositionTypeOrg.class.getSimpleName());
```

Comment:
I try running the application and tests removing these lines and I cannot see any error...
Are you sure that this line is required?